xbmc/utils/DisplayInfo.h

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (C) 2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <memory>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+struct di_info;
+struct di_cta_hdr_static_metadata_block;
+
+namespace KODI
+{
+namespace UTILS
+{
+
+enum class Eotf
+{
+  TRADITIONAL_SDR,
+  TRADITIONAL_HDR,
+  PQ,
+  HLG,
+};
+
+class CDisplayInfo
+{
+public:
+  static std::unique_ptr<CDisplayInfo> Create(const std::vector<uint8_t>& edid);
+
+  ~CDisplayInfo();
+
+  bool SupportsHDRStaticMetadataType1() const;
+
+  bool SupportsEOTF(const Eotf& eotf) const;
```

Comment:
Maybe pass by value as it is only an enum?

Reply:
Ah yes, I meant to do that. Thanks